frappe/integrations/doctype/webhook/webhook.py

```diff
@@ -121,6 +121,7 @@ def log_request(url, headers, data, res):
 	)
 
 	request_log.save(ignore_permissions=True)
+	frappe.db.commit()
```

Comment:
This will be problematic as it commits everything that happened so far and there could be a failure in docevents that proceed this. 

Simplest example: stock/account transactions have various validations on on_submit.

Reply:
IMO a better alternative could be a shorter/frequent queue for deferred insert... was just dealing with another performance issue caused by a barrage of webhook request log (why does this need naming series in the first place?)